CWE/events/Essential Events 8.txt

```diff
@@ -7357,6 +7357,21 @@ country_event = {
 	option = {
 		name = "May we have lasting peace in Europe"
 		add_country_modifier = { name = ger_war_rep duration = 3650 }
+                770 = { remove_core = ITA }
```

Comment:
I prefer to put this as part of the earlier set of events as otherwise the checks from there will have to be moved to this event

Reply:
The better approach to link the peace treaty event and the formation of the free territory of Trieste event is to make the peace treaty even set a global flag which is checked for by the Trieste formation event.